### PR TITLE
PWGCF: flow-tasl: add systematic check options

### PR DESCRIPTION
**Pull request by Luzhiyongg**

### CODE REVIEW

**Comment by Luzhiyongg** [comment]
@victor-gonzalez Hi victor, sorry to bother you in the night. could you please check the update and give the approval? The auto-submission of my target dataset is tomorrow night, I want to get that in time.

File: PWGCF/Flow/Tasks/FlowTask.cxx [hunk 1]
```diff
@@ -545,7 +557,24 @@ struct FlowTask {
...
+    else if (cfgFilterFlag == 1)
+      return (track.isGlobalTrackSDD() == (uint8_t) true);
+    else if (cfgFilterFlag == 2)
+      return (track.isGlobalTrackWoTPCCluster() && track.tpcNClsFound() >= cfgCutTPCclu);
+    else if (cfgFilterFlag == 3)
+      return (track.isGlobalTrackWoPtEta() && (abs(track.eta()) < cfgCutEta) && (track.pt() > cfgCutPtMin) && (track.pt() < cfgCutPtMax));
+    else if (cfgFilterFlag == 4)
+      return (track.isGlobalTrackWoDCA() && track.dcaZ() <= cfgCutDCAz && track.dcaXY() <= cfgCutDCAxy * pow(track.pt(), -1.1));
+    else if (cfgFilterFlag == 5)
+      return (track.isGlobalTrackWoDCATPCCluster() && track.dcaZ() <= cfgCutDCAz && track.dcaXY() <= cfgCutDCAxy * pow(track.pt(), -1.1) && track.tpcNClsFound() >= cfgCutTPCclu);
```
Comment (victor-gonzalez) [inline PWGCF/Flow/Tasks/FlowTask.cxx:573]: For the next iteration I would suggest a `switch` statement. It will be more efficient

**Review by victor-gonzalez — APPROVED** [review]
Have a look at my comment for the next iteration

**Comment by victor-gonzalez** [comment]
Also for the next iteration, please, fix the compiler warnings on your source code
The compiler warnings are going to be hadled as errors once all the compiler warnings are fixed

**Pull request merged**